requirements.txt

```
@@ -0,0 +1,7 @@
+numpy>=1.20.0
+pillow>=8.1.0
+pygame>=2.0.1
+opencv>=4.0.1
```

I created a fresh virtualenvironment with Python 3.8 and when running `pip install -r requirements.txt`, this error occurs:

```
ERROR: Could not find a version that satisfies the requirement opencv>=4.0.1
ERROR: No matching distribution found for opencv>=4.0.1
```

Solution: Package is called `opencv-python`

```suggestion
opencv-python>=4.0.1
```

---

This was automatically created by PyCharm, but you are right
opencv-python is the correct one. Changed it in the last commit.